GitCommands/RevisionReader.cs

```diff
@@ -473,7 +472,7 @@ public StringLineReader(ReadOnlySpan<char> s)
                 return StringPool.Shared.GetOrAdd(_s.Slice(startIndex, lineLength));
             }
 
-            public (string? subject, string? body, bool hasMultiLineMessage) PeakToEnd(bool skipBody)
+            public (string? subject, string? body, bool hasMultiLineMessage) PeakSubjectBody(bool skipBody)
```

Comment:
I am not quite familiar with this code but I think the first word in the method name should be _Pe**e**k_, not _Pe**a**k_. It also occurs in some comments so ideally these should be updated, too.

Reply:
will change